ext/drbbdup/drbbdup.c

```diff
@@ -343,6 +343,16 @@ drbbdup_destroy_manager(void *manager_opaque)
     dr_global_free(manager, sizeof(drbbdup_manager_t));
 }
 
+static bool
```

Comment:
Add a comment that states that this function must always be called prior to inserting cti of drbbdup's dispatcher.

Reply:
Done.